### ISSUE TITLE
Mid-clickable expanded language lists

### ISSUE THREAD

**Issue by comradekingu** [WeblateOrg/weblate#12434]
Papercut issue

Following up in https://github.com/WeblateOrg/weblate/issues/4656 which I suppose details [what files are touched](https://github.com/WeblateOrg/weblate/pull/12426/files).

### Describe the problem

0. Open https://hosted.weblate.org/accounts/profile/#notifications
1. Notice it has expanded languge lists.
2.  Be happy
3. Notice elements are not mid-clickable

### Describe the solution you would like

I want to visit projects on the followed part without following, to dip my toes in.
Do I want to follow, or do I not?

### Describe alternatives you have considered

I thought about having a list of projects that are actively unfollowed.

### Screenshots

![bilde](https://github.com/user-attachments/assets/a2050c2b-011f-4050-b35b-1413b13c0a72)

### Additional context

Some part of this functionality hopefully allows greater control in rooting out malicious/spam projects.

**Comment by nijel** [comment]
I'm not sure if allowing standard middle click is possible here. @meel-hd what do you think?

**Comment by meel-hd** [comment]
Yes, it is possible to allow middle-click functionality using event listeners in JavaScript. You can detect middle-clicks using the `auxclick` event and check if the button property is equal to 1 (which corresponds to the middle mouse button).

**Comment by nijel** [comment]
Yes, I understand you can listen to the event. What @comradekingu probably wanted is to make these links so that middle-click would open up the language in a new tab.

**Comment by meel-hd** [comment]
> middle-click would open up the language in a new tab.

I think @comradekingu is in the `Profile` > `Settings` > `Notifications` tab (watched projects). He wants mid-click to open the project itself not a specific language. And as I said it is possible to add this functionality, of course.

I'll start on that now.

**Comment by meel-hd** [comment]
### Update: 
middle click has some weird behavior, I went with right click instead.

**Comment by nijel** [comment]
Reopening as I'm reverting https://github.com/WeblateOrg/weblate/pull/13309 because of https://github.com/WeblateOrg/weblate/issues/13889.

**Comment by nijel** [comment]
@meel-hd Is the fix in some pull request?

**Comment by meel-hd** [comment]
@nijel  no, I  pushed to the same branch that was used in the original PR #13309 (merged and then reverted). I think I shouldn't have done that?

I should open a new PR.
